views/nexus.pug

```diff
@@ -0,0 +1,13 @@
+html
+  head
+    title Kubercade Nexus
+    style
+      include style.css
+  body
+    #kubercade_nexus
+      h1 Kubercade Nexus
+      h4 Browse Games
+      each gameInfo in gamesList
+        div
+          a(href='/games/' + gameInfo.game)= gameInfo.name
```

Comment:
Eventually you might want a picture, too.

Reply:
Yeah, that's in our project scoping for week 2. Since this list of games on the nexus is created by looping over the config I figured it's will be easy in the future to just add an img or background image once.